jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -218,42 +218,18 @@ AC_DEFUN([OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU],
 
 AC_DEFUN([OPENJ9_CONFIGURE_JITSERVER],
 [
-  AC_MSG_CHECKING([for jitserver])
   AC_ARG_ENABLE([jitserver], [AS_HELP_STRING([--enable-jitserver], [enable JITServer support @<:@disabled@:>@])])
-  OPENJ9_ENABLE_JITSERVER=false
 
+  AC_MSG_CHECKING([for jitserver])
+  OPENJ9_ENABLE_JITSERVER=false
   if test "x$enable_jitserver" = xyes ; then
-    AC_MSG_RESULT([yes (explicitly enabled)])
-
-    if test "x$OPENJDK_TARGET_OS" != xlinux ; then
-      AC_MSG_ERROR([jitserver is unsupported for $OPENJDK_TARGET_OS])
+    if test "x$OPENJDK_TARGET_OS" = xlinux ; then
```

Comment:
I don't understand this, we're building JITServer for plinux.
https://github.com/eclipse/openj9/blob/be03cea0cb7cdd829a61b283d6558b02c542b308/buildenv/jenkins/variables/defaults.yml#L167

Reply:
I have the same question, both xlinux and plinux should be supported

Reply:
The `if` was inverted.

Reply:
The `x` in `"x$OPENJDK_TARGET_OS"` is the idiom to safely handle the case where `$OPENJDK_TARGET_OS` is the empty string. If we didn't have to be concerned about that, the equivalent expression would be: `"$OPENJDK_TARGET_OS" = linux`.

Reply:
got it.